react-components/src/hooks/network/getSourceRelationships.ts

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import {
+  type RelationshipsFilterRequest,
+  type CogniteClient,
+  type CogniteExternalId,
+  type RelationshipResourceType
+} from '@cognite/sdk';
+
+import { filterRelationships } from './filterRelationships';
+import { type ExtendedRelationship } from '../../utilities/types';
+
+type Payload = {
+  targetExternalIds: CogniteExternalId[];
+  sourceTypes?: RelationshipResourceType[];
+} & Omit<RelationshipsFilterRequest, 'sourceExternalIds' | 'targetTypes'>;
+
+export const getSourceRelationships = async (
+  sdk: CogniteClient,
+  payload: Payload
+): Promise<ExtendedRelationship[]> => {
+  return await filterRelationships(sdk, payload).then((relationships) => {
+    return relationships.map((relationship) => ({
```

Comment:
Just a nitpick - I think I'd prefer just awaiting the result from `filterRelationships` and then performing the `map` normally, instead of using a `then`

Reply:
I see you are doing similar things in the other `network` files too

Reply:
oh yeah.. forgot to change that. Agree

https://github.com/cognitedata/reveal/pull/4414/commits/c7f080ddaa9bff8a4c58db3dad5ae0e1492c4d86